### PR TITLE
implement white/blacklist of the backend

### LINKED ISSUE #432: Implement full filter support for the blacklist feature
Users want to blacklist specific items e.g. uncommon enchanted books. For that, the blacklist feature should support the filter feature.

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #432

### CODE REVIEW

**Comment by matthias-luger** [comment]
@Ekwav I think the UI works and sends the correct data but the new backend doesnt work.

**Comment by Ekwav** [comment]
There was indeed a bug when specifying a blacklist with an empty array. 
I fixed it.  
This PR should target `develop` it would break production

File: src/properties.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 let properties = {
-    "commandEndpoint": window.location.host === 'localhost:3000' || window.location.hostname.includes("pr-env-sky-") ? "https://skyblock-backend.coflnet.com/command" : "/command",
-    "apiEndpoint": window.location.host === 'localhost:3000' || window.location.hostname.includes("pr-env-sky-") ? "https://skyblock-backend.coflnet.com/api" : "/api",
-    "websocketEndpoint": "wss://skyblock-backend.coflnet.com/skyblock",
+    "commandEndpoint": window.location.host === 'localhost:3000' || window.location.hostname.includes("pr-env-sky-") ? "https://sky-commands.coflnet.com/command" : "/command",
+    "apiEndpoint": window.location.host === 'localhost:3000' || window.location.hostname.includes("pr-env-sky-") ? "https://sky-commands.coflnet.com/api" : "/api",
+    "websocketEndpoint": "wss://sky-commands.coflnet.com/skyblock",
```
Comment (Ekwav) [inline src/properties.js:4]: This PR should target `develop` it would break production

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Review by Ekwav — CHANGES_REQUESTED** [review]
Some issues I found:
- When any filter is changed all flips are deleted.
- a date filter is displayed as timestamp
- the filters are only for "aspect of the end" it should request "*"

**Pull request merged**